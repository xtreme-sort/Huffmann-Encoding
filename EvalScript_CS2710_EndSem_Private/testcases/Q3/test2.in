GenerateTreeEnhanced
22
6 3
i 4
f 8
z 8
I 10
V 14
7 15
t 15
T 16
k 18
3 20
x 25
h 26
9 30
O 34
a 34
S 35
r 36
Q 38
g 38
4 43
d 45
Quit
