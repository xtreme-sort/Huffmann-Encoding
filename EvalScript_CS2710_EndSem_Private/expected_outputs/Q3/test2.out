(((d,(x,h)),(((V,7),9),((t,((6,i),f)),O))),(((a,(T,k)),(S,r)),((Q,g),(((z,I),3),4))));
